src/services/RetroGenerationService.ts

```diff
@@ -106,30 +117,29 @@ class RetroGenerationService {
                   detailsTemplate.testType.value = (testType.testCode).toUpperCase();
                   detailsTemplate.seatsAndAxles.value = (testResult.vehicleType === VEHICLE_TYPES.PSV) ? testResult.numberOfSeats : testResult.noOfAxles;
                   detailsTemplate.result.value = testType.testResult;
-                  detailsTemplate.certificateNumber.value = testType.certificateNumber;
+                  detailsTemplate.certificateNumber.value = certificateNumber;
                   detailsTemplate.expiryDate.value = testType.testExpiryDate ? moment(testType.testExpiryDate).tz(TimeZone.LONDON).format("DD/MM/YYYY") : "";
                   detailsTemplate.preparerId.value = testResult.preparerId;
                   detailsTemplate.failureAdvisoryItemsQAIComments.value = defects
-                    + reasonForAbandoning
-                    + additionalCommentsAbandon
-                    + LECNotes
-                    + "Additional test type notes: " + additionalTestTypeNotes + ";\r\n"
-                    + (testType.additionalNotesRecorded ? (testType.additionalNotesRecorded + ";") : "");
+                                                                                      + reasonForAbandoning
+                                                                                      + additionalCommentsAbandon
+                                                                                      + LECNotes
+                                                                                      + "Additional test type notes: " + additionalTestTypeNotes + ";\r\n"
+                                                                                      + (testType.additionalNotesRecorded ? (testType.additionalNotesRecorded + ";") : "")
+                                                                                      + customDefectsStr;
```

Comment:
Why the huge indentation?

Reply:
auto-fix indentation is messing with me. fixed